src/utils/helpers/createEvent/index.spec.js

```diff
@@ -0,0 +1,89 @@
+import createEvent from './index';
+
+
+const mockInputTarget = {
+  id: 'username_id',
+  name: 'username',
+  value: 'CarbonUser'
+};
+
+class Event {
+  constructor(target = mockInputTarget) {
+    this.target = target;
+  }
+}
+
```

Comment:
`describe` is used to group a similar set of tests together and allows you to leverage `beforeEach` in this case there is no code shared between tests. The only benefit would be the jest report format, but it would increase the indentation of the entire file. Pros&Cons really. Happy to change it if you would like?